addons/rose/addon/components/rose/code-editor/toolbar/index.hbs

```diff
@@ -0,0 +1,26 @@
+<div class='code-editor-toolbar' data-test-code-editor-toolbar>
+  <div data-test-code-editor-toolbar-action class='code-editor-toolbar__action'>
+    {{yield to='action'}}
+  </div>
+
+  <div class='code-editor-toolbar__right-aligned'>
+    {{#if (has-block 'menu')}}
+      <div data-test-code-editor-toolbar-menu class='code-editor-toolbar__menu'>
+        {{yield to='menu'}}
+      </div>
+      <div
+        class='code-editor-toolbar__menu-divider'
+        data-test-code-editor-toolbar-menu-divider
+      ></div>
+    {{/if}}
+    <CopyButton
```

Comment:
Where is this coming from?

Reply:
This is an import from a new package: [ember-cli-clipboard](https://github.com/jkusa/ember-cli-clipboard)

Reply:
Ah.  As a housecleaning item after this PR is merged, we might update the existing copyable components to use this package instead of clipboard.js.